### PR TITLE
SBT 1.1

### PR DESCRIPTION
**Pull request by danbills**
Basically changed the sbt assembly from "--error" to "early(error)" as they changed the flag.

The other stuff is just 1.X cleanup

### CODE REVIEW

File: centaur/test_cromwell.sh [hunk 1]
```diff
@@ -109,7 +109,7 @@ cd "${RUN_DIR}"
 TEST_STATUS="failed"
 
 ENABLE_COVERAGE=${CENTAUR_SBT_COVERAGE} sbt centaur/it:compile
-CP=$(ENABLE_COVERAGE=${CENTAUR_SBT_COVERAGE} sbt "export centaur/it:dependencyClasspath" --error)
+CP=$(ENABLE_COVERAGE=${CENTAUR_SBT_COVERAGE} sbt "early(error)" "export centaur/it:dependencyClasspath")
```
Comment (mcovarr) [inline centaur/test_cromwell.sh]: the syntax 😦

File: .travis.yml [hunk 2]
```diff
@@ -18,8 +18,8 @@ cache:
     - $HOME/.sbt/boot/
 before_cache:
   # Tricks to avoid unnecessary cache updates
-  - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
-  - find $HOME/.sbt -name "*.lock" -delete
+  - find $HOME/.ivy2/cache -name "ivydata-*.properties" -print -delete
+  - find $HOME/.sbt -name "*.lock" -print -delete
```
Comment (mcovarr) [inline .travis.yml]: why `-print`?

File: wom/src/main/scala/wom/util/JsArray.scala [hunk 3]
```diff
@@ -1,5 +0,0 @@
-package wom.util
-
-
-
-
```
Comment (mcovarr) [inline wom/src/main/scala/wom/util/JsArray.scala]: um what x 3?

File: centaur/test_cromwell.sh (hunk 1 above)
Comment (kshakir) [reply centaur/test_cromwell.sh]: https://github.com/sbt/sbt/issues/4088

File: .travis.yml (hunk 2 above)
Comment (danbills) [reply .travis.yml]: It prints out the files that were deleted.  Not that important, just nice to know that it's working IMO.

File: centaur/test_cromwell.sh (hunk 1 above)
Comment (danbills) [reply centaur/test_cromwell.sh]: For some reason "-error" works, using that to ease the pain.  I think they are undecided on their UNIX-ness vs Java-ness

File: wom/src/main/scala/wom/util/JsArray.scala (hunk 3 above)
Comment (danbills) [reply wom/src/main/scala/wom/util/JsArray.scala]: source detritus

File: build.sbt [hunk 4]
```diff
@@ -279,5 +279,3 @@ lazy val root = (project in file("."))
   .aggregate(wdlTransformsDraft3)
   .aggregate(wom)
   .aggregate(womtool)
-  // TODO: See comment in plugins.sbt regarding SBT 1.x
-  .enablePlugins(CrossPerProjectPlugin)
```
Comment (cjllanwarne) [inline build.sbt]: Is this a real change or just no longer required?

Comment (danbills) [reply build.sbt]: It's part of the sbt "doge" plugin (helps for building against multiple scala versions) that was merged into SBT proper as of 1.x.  Hence the plugin is no longer needed.

**Pull request merged**